libpromises/failsafe.cf

```diff
@@ -252,6 +252,7 @@ body copy_from u_scp(from)
 {
       source      => "$(from)";
       compare     => "digest";
+    trust_server::
```

Comment:
+1 for the trust_server class! Nice way of doing it. I would put a comment here though, explaining how it is set.

(Recommenting in PR instead of in commit)